internal/resource/http.go

```diff
@@ -39,8 +39,8 @@ const (
 	initialBackoff = 100 * time.Millisecond
 	maxBackoff     = 5 * time.Second
 
-	defaultHttpResponseHeaderTimeout = 10
-	defaultHttpTotalTimeout          = 0
+	defaultHttpResponseHeaderTimeout = 5
+	defaultHttpTotalTimeout          = 10
```

Comment:
This _will_ break users. This will fail to fetch anything that cannot be fetched in 10 seconds.

Reply:
I guess since it has been 0, it can never be tightened without potentially breaking someone?

Reply:
correct. If users want to timeout they can set it in the `ignition.timeouts` section